extensions/2.0/Vendor/FB_materials_lightmap/README.md

```diff
@@ -0,0 +1,80 @@
+# FB\_materials\_lightmap
+
+## Contributors
+
+* Alfred Young, Facebook, [@idorurez](https://github.com/idorurez)
+* Pär Winzell, Facebook, [@zellski](https://twitter.com/zellski)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds an additional annotation to a material, providing a means to introduce a lightmap to be applied in the same scope.
+
+Some proven real-world applications for this information:
+1. Having multiple variations of lighting scenarios without having to bake lighting into a single, original base color/texture.
+2. Specify a separate UV channel (texCoord), to vary the new texture texel resolution with respect to the texture it's being applied to.
+3. Modulate the intensity of a lightmap, dimming or increasing it's effect.
+
+## Color
+
+Lightmap texture values use the sRGB color primaries and the sRGB transfer function.
+
+## Specifying Material extension
+
+The data is defined as an optional extension to a glTF `materials`, by adding an `extensions` property and defining a `FB_materials_lightmap` property with a `lightmapTexture` node inside it:
+
+```javascript
+"materials" : [
+  	{
+  		"name" : "material_test",
+  		"pbrMetallicRoughness" : {
+  			"baseColorTexture" : {
+  				"index" : 0,
+  				"texCoord" : 0
+  			}
+  		},
```

Comment:
Should include `KHR_materials_unlit` in the example? Or does FB render unlit even without this?

Reply:
FB renders unlit even without this. As suggested earlier, it may avoid confusion if we specifically name the material based on application, so this may be moot.